src/analysis/timeline/timelineCredView.test.js

```diff
@@ -0,0 +1,109 @@
+// @flow
+
+import {sum} from "d3-array";
+import sortBy from "lodash.sortby";
+import {utcWeek} from "d3-time";
+import {NodeAddress, Graph} from "../../core/graph";
+import {TimelineCred, type TimelineCredConfig} from "./timelineCred";
+import {TimelineCredView} from "./timelineCredView";
+import {type FilteredTimelineCred} from "./filterTimelineCred";
+import {defaultWeights} from "../weights";
+
+describe("src/analysis/timeline/timelineCred", () => {
```

Comment:
sharness is failing because this has the wrong filename.
suggest changing to:

`describe("analysis/timeline/timelineCredView"`